README.md

```diff
@@ -34,6 +34,24 @@ This package provides a KeyShot plugin script that creates jobs for AWS Deadline
         - You can navigate to the folder by going to Finder, clicking the menu for Go -> Go to Folder, and typing in the folder path.
 3. Launch KeyShot. The submitter can be launched within KeyShot from `Window > Scripting Console > Scripts > Submit to AWS Deadline Cloud > Run`
 
+#### Submission Modes
+
+There are two submission modes for the KeyShot submitter which a dialog will ask you to select from before opening the submitter UI.
+
+1. Attach `The scene BIP file and all external files references`
+    - The open scene file and all external files referenced within will be included
+    as job attachments. The submitter will export the open scene to a
+    KSP(KeyShot Package) which turns all file paths in the scene into relative
+    paths and creates a flattened directory with all of the external files directly
+    beside the scene file. The KSP is then unzipped, and the new scene file with
+    the relative paths and all external files will be submitted with the job.
+    The temporary directory used to save the KSP will be deleted after each
```

Comment:
Have we considered not deleting this temp KSP file but saving it with submission timestamp, for example? This way we would create a "snapshot" of the scene for each submission, so it would be easy to jump back to the required revision, if needed. Not sure if there is a real use-case behind it.

Also, if current scene is actually a part of existing KSP file, it seems like we would still create a temp SKP file, update all the paths, create temp directory and delete it. Do we need to have all these steps for existing KSP files?

Reply:
I considered it, but couldn't think of a good enough reason right now to add the extra complexity of managing that. No issue with it being a future improvement if that's a workflow that's requested.

As far as I know if a user tries to open an existing ksp up it has to be extracted by KeyShot somewhere(either the Resources folder or another location) before KeyShot can be used, so everything should be as a bip file during submission